### PR TITLE
Removed GCE mentions.

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

File: docs/_layouts/docs.html [hunk 1]
```diff
@@ -38,10 +38,10 @@
     <hr>
     <h5>Tutorials</h5>
     <ul class="list-unstyled">
-       <li>
+       <!-- li>
```
Comment (jdef) [inline docs/_layouts/docs.html]: shouldn't this just be removed all together?

File: docs/docs/tutorial-gce.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 ---
 title: Mesos-DNS on GCP
+
+redirect_to:
+- http://mesosphere.github.io/mesos-dns/
+
 ---
+<!-- This is deprecated -->
```
Comment (jdef) [inline docs/docs/tutorial-gce.md]: why leave this file here if we're no longer linking to it?

Comment (joel-hamill) [inline docs/docs/tutorial-gce.md]: @jdef There's talk of possibly rewriting the tuturial, without GCE - so left in. Feel free to remove.

Comment (jdef) [inline docs/docs/tutorial-gce.md]: OK we can open a new PR to add links for a proper tutorial. In the meantime
we should avoid leaving comment blocks in the code like this. Please remove
and squash the PR.

Comment (joel-hamill) [inline docs/docs/tutorial-gce.md]: @ssk2 Please make your final decision here. Thanks.

**Comment by ssk2** [comment]
@joel-hamill see #392.

**Pull request closed without merging**